src/pages/App/Booking/BookingsAll.js

```diff
@@ -1,32 +1,46 @@
-import { Button, Container, PageWrapper, Text } from '@qonsoll/react-design'
+import { Button, Container, PageWrapper } from '@qonsoll/react-design'
+import { STATUSES, TIME } from '__constants__'
 
-import { BookingList } from 'domains/Booking/components'
-import { useGetBookings } from '../../../domains/Booking/hooks/get'
+import { BookingsByType } from '../../../domains/Booking/components'
+import moment from 'moment'
 import { useHistory } from 'react-router-dom'
 import { useTranslations } from 'contexts/Translation'
 import { useUser } from 'domains/User/context'
 
+const { BOOKED, FUTURE, CANCELED, PASSED } = STATUSES
+const { CURRENT_DATE_FORMAT } = TIME
+
 const BookingsAll = () => {
   const history = useHistory()
-  const { loading } = useUser()
   const { t } = useTranslations()
+  const { user } = useUser()
+  const currentDateFormatted = moment().format(CURRENT_DATE_FORMAT)
 
-  const [
-    clientFutureBookings,
-    futureBookingsLoading,
-    futureBookingError,
-    clientPassedBookings,
-    passedBookingsLoading,
-    passedBookingError,
-    clientCanceledBookings,
-    canceledBookingsLoading,
-    canceledBookingError
-  ] = useGetBookings()
   const goToNextBookingPage = () => {
     history.push('/next-booking')
   }
-  const computedError =
-    futureBookingError || canceledBookingError || passedBookingError
+
+  const futureBookingsRef = [
+    ['userId', '==', user?._id],
```

Comment:
Don't u think that this row is an exact value of `additionalQuery` variable ?🙂
Figure out somehow rows below as well